assets/js/theme/common/product-details.js

```diff
@@ -595,4 +596,23 @@ export default class ProductDetails {
             $radio.attr('data-state', $radio.prop('checked'));
         });
     }
+
+    /**
+     * Check for fragment identifier in URL requesting a specific tab
+     */
+    getTabRequests() {
+        if (window.location.hash && window.location.hash.indexOf('#tab-similar') === 0 || window.location.hash.indexOf('#tab-warranty') === 0) {
```

Comment:
Can we not have hardcoded check for the hash in here. Bcoz there could be different ones e.g. http://cornerstone-light-demo.mybigcommerce.com/all/able-brewing-system/ is a cornerstone demo store in which we have `#tab-related` which doesn't exist in the check. May be we just check if hash exists & `.tabs` has a href with that hash.

Reply:
@junedkazi `#tab-related` is open by default, as is `#tab-description`, which is why I'm only checking for the others. I guess removing the hardcoded checks would allow for future tabs or custom added ones?

Reply:
Yes it would support any new ones we plan to add in the future or any custom ones which a theme developer will add while working on their custom theme.

Reply:
Are we OK to go with 

```
if (window.location.hash && window.location.hash.indexOf('#tab') === 0) {
```

to ensure we're actually requesting a tab and not, for instance, the reviews panel, i.e. `#product-reviews` ?